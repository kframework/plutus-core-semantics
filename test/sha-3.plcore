(program

(module Foo (imported) (exported () ()) (local () ())
  (define result (builtin sha3_256 #ABCD)))

)
