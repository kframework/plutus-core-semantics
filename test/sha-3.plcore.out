<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> #34bbdf1adad011d1feeb7c17864589b07af0cbc2759b0d402e248f55abc5bad9 </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9999 </gas> </T>
